resources/js/processes/screen-builder/typeDisplay.js

```diff
@@ -1,10 +1,13 @@
-import FileDownload from "./components/file-download";
+import {renderer, FormBuilderControls} from "@processmaker/spark-screen-builder";
+import FileDownload from "./components/file-download.vue";
 
-import {
+const {
     FormMultiColumn,
     FormText,
     FormRecordList
-} from "@processmaker/spark-screen-builder";
+} = renderer;
+
+const TableControl = FormBuilderControls.find(control => control.editorComponent === FormMultiColumn)
 
```

Comment:
The cause of this bug is that the 4 elements using the Display builder were manually defined in spark as a way to limit display builder to just these 4 elements. Because of this, as properties of these elements in screen-builder changed over time that meant these manual ones in spark got stale and broke things.

Rob removed the manual table/multi-form and replaced it with the one in screen-builder.

I believe the right move going forward is to do this for all 4 elements. I would like to discuss this as a group to make sure we have the right assumptions and then split that out into a new ticket.